helm/templates/daemonset.yaml

```diff
@@ -16,8 +18,8 @@ spec:
         app: efs-csi-node
         app.kubernetes.io/name: {{ include "aws-efs-csi-driver.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
-      {{- if .Values.node.podAnnotations }}
-      annotations: {{ toYaml .Values.node.podAnnotations | nindent 8 }}
```

Comment:
If node.podAnnotations and node.tolerations aren't going to be used any longer they should be removed from the values.yaml.

Reply:
Great catch! Will remove. TY